generated/rflx-rflx_utils.adb

```diff
@@ -0,0 +1,63 @@
+pragma Style_Checks ("N3aAbcdefhiIklnOprStux");
+
+package body RFLX.RFLX_Utils with
```

Comment:
How about `RFLX.Arithmetic` or `RFLX.Arith`?

Reply:
I think `RFLX_Arithmetic` would be a good alternative. I will change that.

Reply:
Fixed by fbea48f.